url_launcher/plugin.go

```diff
@@ -0,0 +1,53 @@
+package url_launcher
+
+import (
+	"fmt"
+
+	"github.com/go-flutter-desktop/go-flutter"
+	"github.com/go-flutter-desktop/go-flutter/plugin"
+	"github.com/pkg/browser"
+	"github.com/pkg/errors"
+)
+
+const channelName = "plugins.flutter.io/url_launcher"
+
+type UrlLauncherPlugin struct{}
+
+var _ flutter.Plugin = &UrlLauncherPlugin{} // compile-time type check
+
+func (p *UrlLauncherPlugin) InitPlugin(messenger plugin.BinaryMessenger) error {
+	fmt.Println("InitPlugin")
+	channel := plugin.NewMethodChannel(messenger, channelName, plugin.StandardMethodCodec{})
+	channel.HandleFunc("launch", p.launch)
+	channel.HandleFunc("canLaunch", p.canLaunch)
```

Comment:
The dart plugin can call a method named: [`"closeWebView"`](https://github.com/flutter/plugins/blob/0b8b6f4df57d56d8b86e3c5adcc5a7606d87f06e/packages/url_launcher/lib/url_launcher.dart#L127).
Can you add
```golang
	// Ignored: The plugins doesn't handle WebView. 
	// This call will not do anything, because there is no
	// WebView to closed.
	channel.HandleFunc("closeWebView", func(_ interface{}) (interface{}, error) { return nil, nil })
```